product/index.md

```diff
@@ -2,6 +2,17 @@
 
 This covers how we run our business and deliver service to our customers.
 
+## Product Vision
+Our target customers are users familiar with Node-RED looking to run multiple instances or share access across multiple users.
```

Comment:
```suggestion
FlowForge's target customers are familiar with Node-RED looking to run Node-RED in production with confidence, from single users with a single instance to multiple instances across multiple users.
```

Same vibe, but a consultant starting out alone I think should be included in the target?